vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -293,6 +293,7 @@ class HomeActivityViewModel @AssistedInject constructor(
                 .onEach { status ->
                     when (status) {
                         is SyncRequestState.Idle -> {
+                            checkDeferredPermalink()
```

Comment:
What is preventing this to be executed at each start (i.e. until the Clipboard is flushed?)